_examples/culley-2016.md

```diff
@@ -1,12 +1,45 @@
+  <li>Assessment of current system performance for each point of the exposure space:<ul>
+    <li>Running a simulation model with current operation policy with each climate input (see <a href="../../theory/exploratory-modelling">exploratory modelling</a>)</li>
+    <li>Separating "failure" from "success" conditions based on the definition of thresholds for acceptable values. </li></ul>
+  </li>
+  <li>Assessment of the optimised system performance for each point of the exposure space, i.e. with optimal operation rules for each climate input</li>
+  <li> Superimposition of climate projections onto the exposure space for the current and optimised system performance to identify climate conditions and timing for which the system may fail and when to implement management strategies to maximise a system adaptation capacity (see <a href="../../theory/triggers"> triggers</a>). </li>
+  </ol></li>
+  </ul></p>
+problem_definition: >
+  <p>Lake Como is a regulated lake in Northern Italy with an active storage capacity of 254 Mm<sup>3</sup>, sustaining water needs for irrigation and water supply, together with ensuring flood protection and environmental flow requirements in the Adda River. Its subalpine location implies that the lake is dependent on snowmelt to be filled, which requires the definition of strategic operation rules on an annual basis to support flood control and irrigation water demand objectives. In addition, changes in climate conditions and snowmelt runoff into the lake could increase tensions in the region.
+  </p>
+  <p> The problem is identification of operational rules to maximise the Lake Como system's adaptive capacity under climate change uncertainty</p>
+  <p> Conditions of failure for the system :<ul>
+  <li> daily flooded area over 100 m<sup>2</sup> </li>
+  <li> daily average squared water deficit = 400 kL<sup>2</sup> </li></ul></p>
+  <p>The optimal operation rule is based on a penalty function assessing <a href="../../theory/reliability">reliability</a>:<ul>
+    <li>flood reliability function: the ratio of the days without a flood event within the simulation horizon</li>
+    <li>irrigation reliability function: ratio of available supply to the demand averaged across all time steps, with ratio values capped at 1</li></ul>
+  <p>Climate change projections in the climate exposure space are obtained from multiple models for the years 2025, 2050, 2075 and 2100, with statistical downscaling based on quantile mapping to generate future local climate change projections.  </p>
+  <p> Attributes for generating the climate exposure space:<ul>
+  <li>Mean annual precipitation (mm): Precipitation was perturbed as a percentage change to daily wet days (over 1mm) from 90% to 130% of current values. </li>
+  <li>Mean annual temperature: Temperature was perturbed for each daily time-step from -5℃ to 15℃ of current values. </li></ul></p>
 reference: >
```

Comment:
@Caroline-nc - please do add a figure that demonstrates the method.
Is there also a figure that explains what the operation rule ends up looking like/behaving like?

Reply:
Summary of the approach:
<img src="../../images/culley-2016-approach.jpg" width=50% alt="Outline of the approach to identify the maximum operational adaptive capacity of a water resource system. Four steps are described: 1) generation of an 'exposure space', 2) mapping of the current system on the 'exposure space', 3) identification of the optimal adaptive response to each exposure, 4) superimposition of climate exposure onto the exposure space".>

Illustration of the results for the different climate projections in 2025, 2050, 2075 and 2100:
<img src="../../images/culley-2016-results.jpg" width=50% alt="Climate projections for the years 2025, 2050, 2075 and 2100. The 22 climate scenarios are superimposed on the 'exposure space'. Conditions of success, adaptation and failure for each climate change scenario could be visualised. Eight models (climate change conditions) predict that the system would fail in the future, even under optimised operation conditions".>